pkgs/by-name/km/kmax/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+let
+  pname = "kmax";
+  version = "4.5.1";
+  src = fetchFromGitHub {
+    owner = "paulgazz";
+    repo = "kmax";
+    rev = "v${version}";
+    hash = "sha256-fNs/i8kjc3P6XvX5lvSvNLfhoetr0plNVG+KcRD/Otc=";
+  };
+in
+python3Packages.buildPythonPackage {
+  inherit pname version src;
```

```suggestion
python3Packages.buildPythonPackage rec {
  pname = "kmax";
  version = "4.5.1";

  src = fetchFromGitHub {
    owner = "paulgazz";
    repo = "kmax";
    rev = "v${version}";
    hash = "sha256-fNs/i8kjc3P6XvX5lvSvNLfhoetr0plNVG+KcRD/Otc=";
  };
```
avoiding rec here is rather pointless

---

I will decline here, at least until finalAttrs design pattern be more ubiquitous.